Description: this daemon is usually supervised
Author: A. Schulze
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: smf-spf-2.0.2/smf-spf.c
===================================================================
--- smf-spf-2.0.2.orig/smf-spf.c	2012-09-07 10:19:39.000000000 +0200
+++ smf-spf-2.0.2/smf-spf.c	2012-09-07 10:19:42.000000000 +0200
@@ -950,10 +950,10 @@
 	fprintf(stderr, "smfi_register failed\n");
 	goto done;
     }
-    if (daemon(0, 0)) {
+/*  if (daemon(0, 0)) {
 	fprintf(stderr, "daemonize failed: %s\n", strerror(errno));
 	goto done;
-    }
+    } */
     if (pthread_mutex_init(&cache_mutex, 0)) {
 	fprintf(stderr, "pthread_mutex_init failed\n");
 	goto done;
